pkgs/development/ocaml-modules/alcotest/default.nix

```diff
@@ -11,13 +11,15 @@
   ocaml-syntax-shims,
 }:
 
-buildDunePackage rec {
+buildDunePackage (finalAttrs: {
   pname = "alcotest";
   version = "1.9.1";
 
-  src = fetchurl {
-    url = "https://github.com/mirage/alcotest/releases/download/${version}/alcotest-${version}.tbz";
-    hash = "sha256-HinDtB1DKQYhBbcj39o6/4a4zvXnx1ANDkkfxf145II=";
+  src = fetchFromGitHub {
```

Comment:
`fetchurl` is there for a reason. Please leave it there. You can use `ocamlfind list` to spot the difference between the two.

Reply:
> `fetchurl` is there for a reason. Please leave it there. You can use ocamlfind list to spot the difference between the two.

Could you explain the reason directly instead of making riddles?